src/app/submissions/submissions.component.html

```diff
@@ -40,8 +40,8 @@
       </ng-container>
       <ng-container matColumnDef="user">
         <mat-header-cell *matHeaderCellDef mat-sort-header i18n>Submitted by</mat-header-cell>
-        <mat-cell *matCellDef="let element">
-          <span *ngIf="element.submittedBy !== ''; else unknownLabel">{{element.submittedBy}}</span>
+        <mat-cell *matCellDef="let element" (click)="$event.stopPropagation()">
+          <span *ngIf="element.submittedBy !== ''; else unknownLabel" [routerLink]="['/users/profile/',element.submittedBy]">{{element.submittedBy}}</span>
```

Comment:
Link needs to be submission.user.name if it exist. Else it will not require link.
Submission can be done by
* Submission by valid user logged in
* Submission by custom detail (user does not exist)
* Submission without user detail

Hence only link should be when there is `user.name`

Reply:
Alright. I'll check it out.